format-specs/geoparquet.md

```diff
@@ -50,57 +50,94 @@ Version of the geoparquet spec used, currently 0.3.0
+| orientation   | string              | **OPTIONAL** Winding order of exterior ring of polygons. If present must be 'counterclockwise'; interior rings are wound in opposite order. If absent, no assertions are made regarding the winding order.                                                                                                                                                                                                                                                   |
+| edges         | string              | **OPTIONAL** Name of the coordinate system for the edges. Must be one of 'planar' or 'spherical'. The default value is 'planar'.                                                                                                                                                                                                                                                                                                                             |
+| bbox          | \[number]           | **OPTIONAL** Bounding Box of the geometries in the file, formatted according to [RFC 7946, section 5](https://tools.ietf.org/html/rfc7946#section-5).                                                                                                                                                                                                                                                                                                        |
+| epoch         | double              | **OPTIONAL** Coordinate epoch in case of a dynamic CRS, expressed as a decimal year.
 
 #### crs
 
 The Coordinate Reference System (CRS) is an optional parameter for each geometry column defined in geoparquet format.
 
-The CRS must be provided in [WKT](https://en.wikipedia.org/wiki/Well-known_text_representation_of_coordinate_reference_systems) version 2, also known as **WKT2**. WKT2 has several revisions, this specification only supports [WKT2_2019](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html).
+The CRS must be provided in [PROJJSON](https://proj.org/specifications/projjson.html) format, which is a JSON encoding of [WKT2:2019 / ISO-19162:2019](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html), which itself implements the model of [OGC Topic 2: Referencing by coordinates abstract specification / ISO-19111:2019](http://docs.opengeospatial.org/as/18-005r4/18-005r4.html).  Apart from the difference of encodings, the semantics is intended to be exactly the same as WKT2:2019, and PROJJSON can be morphed losslessly from/into WKT2:2019.
```

Comment:
Note there was a comment on this in #97 - https://github.com/opengeospatial/geoparquet/pull/97#discussion_r873977007  That's the 'require' PR, but this one should merge first so we can cut 0.4.0 release.

Reply:
Either one could theoretically go out as 0.4.0; per the discussion I thought the intent was to compare the optional PROJJSON with OGC:CRS84 default vs required PROJJSON, rather than them being additive.  But I can certainly update this PR to address comments in #97 if we'd like to see this particular PR go out sooner.

Reply:
Sounds good. I think the plan was to aim for 0.4.0 without it - I organized [the 0.4 milestone](https://github.com/opengeospatial/geoparquet/milestone/6) around that - and then to be sure to solve this for 0.5. Just to be sure to make iterative progress as it seemed like that one could take longer. 

I think we targeted next week for the 0.4.0 release. I'm also not opposed to get this into 0.4 release with everyone happy, and just put up a discussion in #98 to help push it forward. So maybe see where that discussion is at on monday of next week, and if it's not fully resolved then it'd be great if you could clean up this PR and we push out 0.4